apps/vc-api/src/setup.ts

```diff
@@ -20,13 +20,16 @@ import { NestFactory } from '@nestjs/core';
 import { DocumentBuilder, OpenAPIObject, SwaggerModule } from '@nestjs/swagger';
 import { AppModule } from './app.module';
 
+export const API_DEFAULT_VERSION_PREFIX: string = '/v1';
+export const API_DEFAULT_VERSION: string = API_DEFAULT_VERSION_PREFIX.replace(/^\/v/, '');
```

Comment:
it would be more clear to do the opposite:
```suggestion
export const API_DEFAULT_VERSION: string = 1;
export const API_DEFAULT_VERSION_PREFIX: string = `/v${API_DEFAULT_VERSION}`;
```

Reply:
yeah, good point, thanks Dmitry!